README.md

```diff
@@ -76,7 +73,7 @@ Producer
 ```golang
 import (
 	"fmt"
-	"gopkg.in/confluentinc/confluent-kafka-go.v1/kafka"
+	"github.com/confluentinc/confluent-kafka-go/kafka"
```

Comment:
As mentioned in the previous comments; it is too early to assume the user is using Go modules and have that as the default. When not using go modules this import will reference the master branch, which is dangerous.